src/app/(nextjs_migration)/(guest)/page.tsx

```diff
@@ -18,7 +18,7 @@ export default async function Home() {
 
   return (
     <div data-partial="landing">
-      <Script type="module" src="/nextjs_migration/client/js/transitionObserver.js" />
+      <script type="module" src="/nextjs_migration/client/js/transitionObserver.js" async></script>
```

Comment:
Any reason you didn't go for the [`lazyOnload` strategy](https://nextjs.org/docs/pages/building-your-application/optimizing/scripts#strategy) for these?